doc/developer/material.md

```diff
@@ -407,23 +407,43 @@ builder.second( rt, isMaterialTransparent );
 
 
 ## Rendering without using Materials {#non-bsdf-rendering}
-The _Radium Material Library_ and related components are mainly designed to manage Materials as a representation of a 
-_Bidirectional Scattering Distribution function (BSDF)_.
+The _Radium Material Library_ and related components are mainly designed to manage Materials as a representation of a _Bidirectional Scattering Distribution function (BSDF)_.
 
-When rendering, it is sometime useful to compute the final color of an object that do not rely on a bsdf but just 
-on a specific color for each geometry fragment.
+When rendering, it is sometime useful to compute the final color of an object that do not rely on a bsdf but just on a specific color for each geometry fragment.
 
-Even if the Ra::Engine::PlainMaterial could be used for this and assuming that such a way to define material is object 
-specific, the following steps are required :
+Even if the Ra::Engine::PlainMaterial could be used for this and assuming that such a way to define material is object specific, the following steps are required :
```

Comment:
It is not clear to me if I can use `PlainMaterial` for this, or not.
Do you mean that it is not recommended ? or that it cannot be used ?

Reply:
I add this line because i had the following question from another reviewer : "Why allowing to render without material. I can do the example using PlainMaterial" ...  But doing such is an ugly hack even if the following example is similar in its computation and limitation than the plain material. Feel free to propose the sentence that will make clear that such hacks must be prevented.

Reply:
I'm not sure to catch the point here, what is the good way for one to have some specific rendering computation.
Let's say I'm doing a display shader to display an hdr texture on a quad for debug purposes. 
I want to control some simple parameter like color scale.
Is it a good idea to define a material ? or should I use the materialess approach explained here ?

Reply:
(and I do not find the reference to the other reviewer question in history ... if you have a link it could help to understand the changes).